testutil/testutil.go

```diff
@@ -76,6 +77,30 @@ func Equals(tb testing.TB, exp, act interface{}, v ...interface{}) {
 	tb.Fatal(sprintfWithLimit("\033[31m%s:%d:"+msg+"\n\n\texp: %#v\n\n\tgot: %#v%s\033[39m\n\n", filepath.Base(file), line, exp, act, diff(exp, act)))
 }
 
+type goCmp struct {
+	opts cmp.Options
+}
+
+func WithGoCmp(opts ...cmp.Option) goCmp {
```

Comment:
Let's add comment.

Reply:
Let's also mention in the comment that compatibility guarantee for this function arguments is same as go-cmp, so... zero as it is at  version 0.x. It's fine, but let's just mention it in comment